utils/utils_test.go

```diff
@@ -98,12 +98,34 @@ func TestGetIpAddress(t *testing.T) {
 		RemoteAddr: "10.2.0.1:12345",
 	}
 
-	assert.Equal(t, "10.1.0.1", GetIpAddress(&httpRequest4))
+	assert.Equal(t, "10.1.0.1", GetIpAddress(&httpRequest4, []string{"X-Forwarded-For", "X-Real-Ip"}))
 
 	// Test without any headers
 	httpRequest5 := http.Request{
 		RemoteAddr: "10.2.0.1:12345",
 	}
 
-	assert.Equal(t, "10.2.0.1", GetIpAddress(&httpRequest5))
+	assert.Equal(t, "10.2.0.1", GetIpAddress(&httpRequest5, []string{"X-Forwarded-For", "X-Real-Ip"}))
+
+	// Test with both headers, but both untrusted
+	httpRequest6 := http.Request{
+		Header: http.Header{
+			"X-Forwarded-For": []string{"10.3.0.1"},
```

Comment:
Can we add some coverage for the comma-encoded versions of this header?

Reply:
It's done in https://github.com/mattermost/mattermost-server/pull/10907/files/0f2d1d71c4c50fe3b53d63005fcb9a436594c225#diff-8d029438ee1ad6087ae32de5b0636ccbR74 - Is there something you'd like to see additionally?

Reply:
Sorry, understood that you've meant for the multiple version, but untrusted and trusted. Added in the newest commit.